src/sentry/static/sentry/app/routes.jsx

```diff
@@ -37,10 +37,10 @@ const lazyLoad = cb => m => cb(null, m.default);
 
 const hook = name => HookStore.get(name).map(cb => cb());
 
-const OrganizationMembersView = HookOrDefault({
+const OrganizationMembersWrapper = HookOrDefault({
   hookName: 'component:org-members-view',
   defaultComponentPromise: () =>
-    import(/* webpackChunkName: "OrganizationMembers" */ 'app/views/settings/organizationMembers'),
+    import(/* webpackChunkName: "OrganizationMembersWrapper" */ 'app/views/settings/organizationMembers/organizationMembersWrapper'),
```

Comment:
I guess we don't _have_ to immediately update the associated getsentry thing

Reply:
Otherwise the wrapper is visible along with the hooked component

Reply:
Ah yeah, but prob fine

Reply:
Though I already see the PR update :)